### PR TITLE
introduced functions

### PR DESCRIPTION
**Pull request by Tanner253**
TAKE THIS

### CODE REVIEW

File: js/app.js [hunk 1]
```diff
@@ -6,78 +6,92 @@ var answers = [];
 var correctAnswers = 0;
 var questionChances = [4, 6]
 
-userName = prompt('First, please tell me your name');
-console.log('User Name:', userName);
 
-answers[1] = prompt('Hi ' + userName + '!  Nice to meet ya!  We\'ll start with an easy one: Is my last name ROSKA?');
-answers[1] = answers[1].toUpperCase();
-console.log('Last Name answer:', answers[1]);
+    var userName = prompt('First, please tell me your name');
```
Comment (ARoska) [inline js/app.js:10]: please correct the indentation

Comment (ARoska) [inline js/app.js:10]: please correct this indentation

**Review by ARoska — CHANGES_REQUESTED** [review]
please correct the indentation on lines 13 through 16

File: js/app.js (hunk 1 above)
Comment (Tanner253) [reply js/app.js:10]: OKAY

**Pull request merged**